src/reference/sectionC_interpretation.inc

```diff
@@ -186,14 +186,24 @@ Interpretation of ``units`` elements
 
       Tuples for each :code:`unit` child element SHALL be determined as follows:
 
+      .. _3_3_3_2_1:
```

Comment:
Is this for the hardcoded refs? Should we avoid having the section number in there then? Or is this the name that's displayed?

Reply:
nothing is displayed, it's just where the link points to